pkg/catalog/manager/traverse.go

```diff
@@ -133,8 +141,8 @@ func (m *Manager) traverseAndUpdate(helm *helmlib.Helm, commit string, cmt *Cata
 			v.VersionName = version.Name
 			v.VersionURLs = version.URLs
 
-			v.UpgradeVersionLinks = map[string]string{}
 			for _, versionSpec := range template.Spec.Versions {
+				v.UpgradeVersionLinks = map[string]string{}
```

Comment:
don't you only want to do this map is null? otherwise it's being rewritten

Reply:
I decided to keep the original logic, just move it into the method that actually sets that value. Since we are building the value,  v.UpgradeVersionLinks will always be null until we set it so doing a null check was not needed.

Reply:
wait, yes, you are correct - I will add the null check back in!